src/alitra/align_frames.py

```diff
@@ -18,14 +18,14 @@ def align_frames(
     ) -> FrameTransform:
         """Let p_1 be fixed local coordinate frame that the robot operate in, and p_2 be
         in the asset-fixed global coordinate system. Further, let the relationship between the two
-        reference systems be described in the _to frame be : p_1 = rotation_object(euler,p_2) + translations,
-        using the zyx Euler angle rotation convention. This function finds the rotations and translations
-        by matching the two coordinate systems, and represent the transformation through a CoordinateFrame object.
+        reference systems be described in the _to frame be : p_1 = rotation_object.apply(p_2) + translation,
+        This function finds the rotation_object and translation by matching the two coordinate systems,
+        and represent the transformation through a CoordinateFrame object.
```

Comment:
```suggestion
        and represent the transformation through a FrameTransform object.
```
Should it be this instead? I don't recall seeing a reference to CoordinateFrame anywhere.